src/main/org/codehaus/groovy/control/StaticImportVisitor.java

```diff
@@ -317,6 +337,10 @@ protected Expression transformMethodCallExpression(MethodCallExpression mce) {
         return result;
     }
 
+    private boolean isInnerClass(ClassNode currentClass) {

```

Comment:
There was an identical method added to `StaticVerifier`, how about replacing both with a static method in `ClassNodeUtils`?